src/backend/gporca/libnaucrates/src/operators/CDXLTableDescr.cpp

```diff
@@ -198,6 +215,12 @@ CDXLTableDescr::SerializeToDXL(CXMLSerializer *xml_serializer) const
 	xml_serializer->AddAttribute(CDXLTokens::GetDXLTokenStr(EdxltokenTableName),
 								 m_mdname->GetMDName());
 
+	if (NULL != m_mdalias)
```

Comment:
In 7X aliases are used in place of table names. Such behaviour affects all the serialization procedures, including not only DXL representation, but also various operator tress, memo contents, etc.
For example:
```
explain (costs off) select sum(t2.a), t1.a, t1.b, t1.c from funcdep1 t1 join funcdep2 t2 on t1.b = t2.b group by t1.a;
Algebrized preprocessed query:
+--CLogicalGbAgg( Global ) Grp Cols: ["a" (0), "b" (1), "c" (2)][Global], Minimal Grp Cols: [], Generates Duplicates :[ 0 ] 
   |--CLogicalNAryJoin
   |  |--CLogicalGet "t1" ("t1"), Columns: ["a" (0), "b" (1), "c" (2), "d" (3), "ctid" (4), "xmin" (5), "cmin" (6), "xmax" (7), "cmax" (8), "tableoid" (9), "gp_segment_id" (10)] Key sets: {[0], [4,10]}
   |  |--CLogicalGet "t2" ("t2"),
....
....
Or CMemo contents:
Group 1 (#GExprs: 2):
  0: CLogicalGet "t2" ("t2"), Columns: ["a" (11), "b" (12), "c" (13), "d" (14), "ctid" (15), "xmin" (16), "cmin" (17), "xmax" (18), "cmax" (19), "tableoid" (20), "gp_segment_id" (21)] Key sets: {[4,10]} [ ]
  1: CPhysicalTableScan "t2" ("t2")
```
The question we need to answer: Do we need such behaviour for all the serializable objects, or putting aliases only to DXL is enough for us? We need to clarify that moment.